rust/operator-binary/src/superset_controller.rs

```diff
@@ -668,14 +643,14 @@ fn build_node_rolegroup_service(
             cluster_ip: Some("None".to_string()),
             ports: Some(vec![
                 ServicePort {
-                    name: Some("http".to_string()),
+                    name: Some(APP_PORT_NAME.into()),
```

Comment:
These 3 lines could also be consistent with the `ListenerPort` suggestion further down (that way, we can use `"TCP".to_owned()`).

```rust
        name: Some(APP_PORT_NAME.to_owned()),
        port: APP_PORT.into(),
        protocol: Some("TCP".to_owned()),
```

Reply:
80a5fcf